M/ModuleInterfaceTools/Compat.toml

```diff
@@ -6,3 +6,6 @@ julia = "0.6-1.3"
 
 ["0.2"]
 julia = "1.0-1.3"
+
+["1.0"]
+julia = ["1.0.0-1"]
```

Comment:
https://github.com/JuliaRegistries/General/pull/111#discussion_r277135089

Reply:
That must be coming from JuliaRegistrator, right?

Reply:
Yes.

Reply:
Has that bug already been fixed?

Reply:
I don't think so, you can try retriggering though.

Reply:
Still broken, unfortunately.